data/maps/BattleFrontier_BattlePikeLobby/scripts.inc

```diff
@@ -31,7 +31,7 @@ BattleFrontier_BattlePikeLobby_EventScript_QuitWithoutSaving::
 	lockall
 	msgbox BattleFrontier_BattlePikeLobby_Text_FailedToSaveBeforeQuitting, MSGBOX_DEFAULT
 	closemessage
-	pike_set PIKE_DATA_WIN_STREAK 0
+	pike_set PIKE_DATA_WIN_STREAK, 0
```

Comment:
Missing commas are arguably errors, not just formatting inconsistencies! If the `PIKE_DATA_WIN_STREAK` #define was ever changed to something other than a plain numeric constant, for this example, it's conceivable that the change could trigger a syntax error here. Not that this is likely to happen...but I think it's better to be safe with these things.